adaptors/fhir-4.md

```diff
@@ -0,0 +1,710 @@
+---
+title: FHIR r4 Adaptor
+---
+
+Language support for FHIR version 4. This adaptor is compatible with any FHIR r4
```

Comment:
@josephjclark so this definitely won't work with other FHIR versions, yeah? If so, wondering if we should add another line to the intro - something like: "If you are seeking to integration with another version of FHIR, check out our generic `fhir` adaptor..."

Reply:
Yeah good point. We might be able to add some  validation within the adaptor too